### PR TITLE
Milestone 1: Add Header

### PR DESCRIPTION
**Pull request by BenMukebo**
hhdhdhdd

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,9 +10,9 @@
 <div class="body">
     <!-- <h1>Hello Wordl! we are software developers!</h1> -->
     <header id="main">
-        <h1 id="title">Dr. Ben And Dr. Alick</h1>
+        <h1 id="title">Dr... Ben And Dr. Alick</h1>
```
Comment (barackm) [inline index.html:13]: You don't need to change this line

File: index.html [hunk 2]
```diff
@@ -10,9 +10,9 @@
...
         <p>The man who saved a billion lives</p>
-        
+        <h2>hello from </h2>
         <div id="img-div">
             <img src="img1.jpg" id="image" alt="" srcset="">
```
Comment (barackm) [inline index.html:17]: This is not good

Comment (barackm) [inline index.html:17]: This is not good

**Review by barackm — CHANGES_REQUESTED** [review]
## **STATUS: changes requested !!!!**

take a look at my comments and see how to improve the project.

**Review by barackm — APPROVED** [review]
## **STATUS: APPROVED**

Congratulations on working on the requested changes, you can now merge the changes

File: index.html (hunk 1 above)
Comment (BenMukebo) [reply index.html:13]: please help me understand this bvetter

**Pull request merged**